Controller/Annotations/RequestParam.php

```diff
@@ -19,11 +19,12 @@
  * @Annotation
  * @NamedArgumentConstructor
  * @Target("METHOD")
+ * @Target("CLASS")
```

Comment:
You need to add this to the existing `@Target` annotation to fix the test errors (same for the `FileParam` class).  Double-check this against the `QueryParam` class' annotations.

Reply:
Some habit are easier to forget than others... :grin: 

Thank you.